custodia/message/kem.py

```diff
@@ -373,7 +373,8 @@ def test_1_Parse_GET(self):
         kem = KEMHandler({'KEMKeysStore': self.kk})
         kem.parse(jtok, "mykey")
         out = kem.reply('output')
-        jtok = JWT(jwt=json_decode(out)['value'])
+        msg = json_decode(json_encode(out))
```

Comment:
correct, that is my intention. Your tone sounds like you don't favor the approach